test/quantization/core/test_quantized_op.py

```diff
@@ -4188,9 +4181,9 @@ def test_qlinear_pt2e(self):
         post_op_args = []
         cases = itertools.product(
             in_channels_list, out_channels_list, use_bias_list,
-            supported_post_ops, weight_quant_per_channel_list, output_dtype_list)
+            supported_post_ops, weight_quant_per_channel_list, output_dtype_list, post_op_algorithm)
```

Comment:
Make it plural and make it a list.
```suggestion
            supported_post_ops, weight_quant_per_channel_list, output_dtype_list, post_op_algorithms)
```